src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,65 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+import React from 'react';
+
+interface Props {
+  id: string,
+  title: string,
+  content: string,
+}
+
+interface TabsType {
+  tabs: Props[],
+  selectedTabId: string,
+  onTabSelected: (value: Props) => void,
+}
```

Comment:
```suggestion

interface Props {
  tabs: Tab[],
  selectedTabId: string,
  onTabSelected: (value: Tab) => void,
}
```
Better create separate interface Tab and import it to App and to here